crates/rethnet_evm/tests/blockchain.rs

```diff
@@ -144,10 +144,10 @@ fn create_dummy_transaction() -> SignedTransaction {
         chain_id: 1,
     };
 
-    let private_key = rethnet_eth::secp256k1::SecretKey::from_str(DUMMY_PRIVATE_KEY)
-        .expect("Failed to parse private key");
+    let secret_key = rethnet_eth::signature::secret_key_from_str(DUMMY_PRIVATE_KEY)
```

Comment:
`DUMMY_PRIVATE_KEY`

Reply:
Done in [e3ec175](https://github.com/NomicFoundation/hardhat/pull/4465/commits/e3ec175495c36b67645a5e9aba0d3151eb910729)